### ISSUE TITLE
[Narrator] AutoSuggestBox drop-down results do not work

### ISSUE THREAD

**Issue by jayrenn** [winjs/winjs#1351]
Narrator does not always work with the drop-down results. Over 90% of the time it gets caught repeating the query text over and over. In the image below, moving up and down the results just keeps repeating “Capital S”.
![image](https://cloud.githubusercontent.com/assets/7266075/9264428/43c4e964-41db-11e5-89d3-786d77c43a50.png)

**Comment by jseanxu** [comment]
Looks like you were in character-mode in narrator while the suggestions are items. If you swipe up a few times till it says "items" you should be able to navigate the control.
